css/style.css

```diff
@@ -1 +1,1461 @@
+  line-height: 44px;
+  margin-bottom: 4px;
+}
+
+.blogpage-card-text {
+  font-family: var(--text-font);
+  color: var(--text-color);
+  font-size: 20px;
+  line-height: 30px;
+  margin-bottom: 20px;
+}
+
+.blogpage-card-readmore {
+  padding: 0 8px;
+  font-size: 16px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  gap: 4px;
+}
+
+.blogpage-card-readmore svg {
+  width: 19px;
+  height: 12px;
+}
+
+.blog-pagination {
+  background-color: var(--main-background-color);
+  padding-bottom: 200px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 16px;
+}
+
+.blog-pagination a {
+  display: block;
+  width: 48px;
+  height: 48px;
+  font-size: 20px;
+  line-height: 48px;
+  font-family: var(--text-font);
+  border: 1px solid var(--secondary-color);
+  padding: 1.5px;
+  text-align: center;
+}
+
+.blog-pagination a.active {
+  background-color: var(--secondary-color);
+  color: var(--btn-primary-color);
+}
+
+/*******************  Blog Article Page ***************************/
+
+.blog-article {
+  padding-top: 100px;
+  text-align: left;
+  background-color: var(--main-background-color);
+}
+
+.blog-article-title {
+  margin-top: 30px;
+  font-size: 36px;
+  line-height: 44px;
+  font-family: var(--btn-header-font);
+}
+
+.blogarticle-data {
+  margin-top: 40px;
+  margin-bottom: 20px;
+  display: flex;
+  justify-content: start;
+  font-size: 20px;
+  line-height: 30px;
+  font-family: var(--text-font);
+  color: var(--text-color);
+}
+
+.blogarticle-data li {
+  padding-right: 12px;
+}
+
+.blogarticle-date-item {
+  padding-left: 12px;
+  border-left: 1px solid var(--text-color);
+}
+
+.blogarticle-data svg {
+  width: 24px;
+  height: 24px;
+  margin-right: 6px;
+}
+
+.blog-story {
+  font-size: 20px;
+  line-height: 30px;
+  font-family: var(--text-font);
+  color: var(--text-color);
+}
+
+.blog-story-quote {
+  margin-top: 24px;
+  margin-bottom: 40px;
+  padding: 26px 32px;
+  border-left: 8px solid var(--btn-primary-color);
+  background-color: rgba(33, 231, 134, 0.1);
+}
+
+.blog-story-quote blockquote {
+  margin-bottom: 12px;
+  font-family: var(--text-font);
+  font-size: 24px;
+  line-height: 30px;
+}
+
+.blog-story-quote figcaption {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+}
+
+.blog-article-rating {
+  font-family: var(--text-font);
+  font-size: 14px;
+  line-height: 22px;
+  color: var(--text-color);
+  display: flex;
+  align-items: center;
+  gap: 4px;
+}
+
+.rating-star {
+  width: 18px;
+  height: 18px;
+}
+
+.blog-story-quote cite {
+  font-size: 16px;
+  line-height: 24px;
+  font-style: normal;
+}
+
+.blog-sub-article {
+  margin-bottom: 20px;
+}
+
+.article-block-title {
+  margin-bottom: 12px;
+  font-size: 30px;
+  line-height: 42px;
+  font-family: var(--btn-header-font);
+}
+
+.article-block-text {
+  font-family: var(--text-font);
+  color: var(--text-color);
+  font-size: 20px;
+  line-height: 30px;
+}
+
+.article-block-title + p + p {
```

Comment:
If you have more that two paragraphs?))
<img width="1463" alt="Screenshot 2024-08-10 at 00 03 46" src="https://github.com/user-attachments/assets/7bcd6792-b0b5-4de2-ac28-b41ee88f494e">

You can use `.article-block-text:not(:first-of-type)` instead

Reply:
#fixed